### PR TITLE
spnet.ru

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

File: src/chrome/content/rules/spnet.ru.xml [hunk 1]
```diff
@@ -1,24 +1,257 @@
...
+	<target host="pbx.spnet.ru" />
 	<target host="pop3.spnet.ru" />
 	<target host="pop3s.spnet.ru" />
 	<target host="smtp.spnet.ru" />
 	<target host="wiki.spnet.ru" />
+	<target host="zabbix.spnet.ru" />
+
+	<target host="www.spnet.ru" />
+
+	<rule from="^http://www\.spnet\.ru/"
```
Comment (J0WI) [inline src/chrome/content/rules/spnet.ru.xml]: Please add also a comment about the issue

File: src/chrome/content/rules/spnet.ru.xml [hunk 2]
```diff
@@ -1,24 +1,257 @@
...
+82.spnet.ru ¹
+846.spnet.ru ¹
+85.spnet.ru ¹
+89-222-212-0.spnet.ru ¹
+89-222-212-1.spnet.ru ¹
+89-222-212-2.spnet.ru ¹
+89-222-212-4.spnet.ru ¹
+89-222-212-5.spnet.ru ¹
+89-222-212-6.spnet.ru ¹
+89-222-212-7.spnet.ru ¹
```
Comment (J0WI) [inline src/chrome/content/rules/spnet.ru.xml]: Well, you don't need to list all the reverse IP hosts. ;)

Comment (galeksandrp) [inline src/chrome/content/rules/spnet.ru.xml]: For the luck, they are using wildcard domain *.spnet.ru (in addition to any-other-static-domains.spnet.ru), which cert is for hosting.spnet.ru only.

File: src/chrome/content/rules/spnet.ru.xml (hunk 1 above)
Comment (J0WI) [inline src/chrome/content/rules/spnet.ru.xml]: @galeksandrp I'm still missing this

Comment (galeksandrp) [inline src/chrome/content/rules/spnet.ru.xml]: This is included in `*.spnet.ru`

**Pull request merged**